src/platforms/platformsConfig.json

```diff
@@ -2,7 +2,7 @@
     "ios": {
         "hostos": ["darwin"],
         "url": "https://github.com/apache/cordova-ios.git",
-        "version": "^5.0.0",
+        "version": "^6.0.0-nightly.2020.5.11.e65d685c",
```

Comment:
This should just be `^6.0.0`, right? Not nightly?

Reply:
Ah yes, that can be 6 now.